$NetBSD: patch-ai,v 1.1 2007/01/22 17:41:17 dmcmahill Exp $

fix prototype to match what the function actually is.  Without this
linking fails on sunpro

--- src/incident.h.orig	1998-12-07 03:28:51.000000000 -0500
+++ src/incident.h
@@ -1,3 +1,3 @@
 complex **Zequ(Conductor *c,int numsgn,int Nc);
-complex **ysort(int Nc,int n,int m);
+void ysort(int Nc,int n,int m);
 
